app/components/blacklight/search_bar_component.rb

```diff
@@ -60,6 +60,12 @@ def advanced_search_enabled?
       blacklight_config.advanced_search.enabled
     end
 
+    def rounded_border_class
+      return 'rounded-0' if search_fields.length > 1
+
+      'rounded-left rounded-start'
```

Comment:
Isn't rounded-left a bootstrap 4 class?  The main branch no longer has support for Bootstrap 4.